apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.ejs

```diff
@@ -340,6 +340,14 @@
       </a>
       <%- include('../../partials/assessmentSyncErrorsAndWarnings'); %>
 
+      <form name="start-bot-grading" method="POST">
+        <input type="hidden" name="__action" value="bot_grade_assessment" />
+        <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+        <div class="mt-4">
+          <button type="submit" class="btn btn-primary">Bot Grade!</button>
+        </div>
+      </form>
+
```

Comment:
Can you move this to a button in the bootstrap-table interface, to make this more consistent with the remaining functionality? You'd need to add an entry to the [`buttons` dictionary](https://bootstrap-table.com/docs/api/table-options/#buttons) in lines 37 and onwards.

Reply:
Also, you probably should only show this button if the bot-grading feature is enabled.

Reply:
So far I've successfully put a button there. But what should I put in the event (or html?) to send the post request?

Reply:
Actually, I think I figured it out